pkg/controller/oneagent/istio.go

```diff
@@ -13,13 +14,24 @@ import (
 	"k8s.io/apimachinery/pkg/labels"
 	"k8s.io/apimachinery/pkg/runtime/schema"
 	"sigs.k8s.io/controller-runtime/pkg/client"
+	"sigs.k8s.io/controller-runtime/pkg/client/config"
 	"sigs.k8s.io/controller-runtime/pkg/controller/controllerutil"
 )
 
 func (r *ReconcileOneAgent) reconcileIstio(logger logr.Logger, instance *dynatracev1alpha1.OneAgent, dtc dtclient.Client) error {
 	var err error
 
-	// TODO determine if cluster runs istio
+	cfg, err := config.GetConfig()
```

Comment:
When the ReconcileOneAgent object is created on newReconciler(), the Manager provides some objects. Get the config from there, e.g., `mgr.GetConfig()`